test/mock/adapter/adapter.go

```diff
@@ -149,6 +150,8 @@ func startAdapterApp(agent *didComm, router *mux.Router) error {
 	router.HandleFunc("/verifier/oidc", app.oidcVerifier)
 	router.HandleFunc("/verifier/oidc/share", app.oidcShare)
 	router.HandleFunc("/verifier/oidc/share/cb", app.oidcShareCallback)
+	router.HandleFunc("/verifier/openid4vc", app.openid4vcVerifier)
```

Comment:
Can we update to `openid4vp` as this is a presentation flow ? Same comments for other instances